src/test/java/uk/gov/justice/hmpps/prison/service/OffenderIdentityServiceTest.kt

```diff
@@ -0,0 +1,100 @@
+package uk.gov.justice.hmpps.prison.service
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.any
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import uk.gov.justice.hmpps.prison.repository.jpa.model.Offender
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderBooking
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderIdentifier
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderIdentifierRepository
+import java.util.*
```

Comment:
As you won't have noticed in our undocumented style guide that doesn't exist.... we try to avoid `*` imports as they're frowned upon nowadays. There's probably a setting in your IDE to control this.